data/i18n/ar/ar.toml

```diff
@@ -48,10 +48,10 @@ other = "و"
 other = "رمز تعريف الCVE"
 
 [cve_issue_url]
-other = "رابط القضية للCVE"
+other = "رابط البلاغ للCVE"
 
 [cve_summary]
-other = "ملخص القضية"
+other = "ملخص البلاغ"
```

Comment:
I don't outright object, but something to bear in mind: this is really summarizing a vulnerability rather than an issue.